docs/introduction/Car-registry-tutorial.rst

```diff
@@ -151,7 +156,7 @@ Transaction to initiate the selling process of the car.
 
         *Output boxes:*
          
-            CarSellOrderBox, which represents the car to be sold, that box could be opened by the initial car owner or specified buyer in case if a buyer buys that car.    
+            CarSellOrderBox, which represents the car to be sold. This box could be opened by the initial car owner to recall the order, or a specified buyer if a buyer buys the car.    
```

Comment:
A CarSellOrderBox represents a car to be sold. This box could be opened by the car owner to recall the order, or by a specified buyer if a someone buys the car.